index.bs

```diff
@@ -9105,6 +9145,64 @@ This section defines a number of
 whose presence affects the ECMAScript binding.
 
 
+<h4 id="AllowResizable" extended-attribute lt="AllowResizable">[AllowResizable]</h4>
+
+If the [{{AllowResizable}}] [=extended attribute=] appears on one of the [=buffer source types=] without the presence of the [{{AllowShared}}] [=extended attribute=], it
+creates a new IDL type that allows the buffer source type to be backed by an ECMAScript
+{{ECMAScript/ArrayBuffer}} that is resizable, instead of only by a fixed-length {{ECMAScript/ArrayBuffer}}.
+
+If the [{{AllowResizable}}] [=extended attribute=] and the [{{AllowShared}}] [=extended attribute=] both appear on one of the [=buffer source types=], it
+creates a new IDL type that allows the buffer source type to be additionally backed by an ECMAScript
+{{ECMAScript/SharedArrayBuffer}} that is growable.
+
+The [{{AllowResizable}}] extended attribute must [=takes no arguments|take no arguments=].
+
+A type that is not a [=buffer source type=] must not be
+[=extended attributes associated with|associated with=] the [{{AllowResizable}}] extended attribute.
+
+See the rules for converting ECMAScript values to IDL [=buffer source types=] in
+[[#es-buffer-source-types]] for the specific requirements that the use of [{{AllowResizable}}] entails.
+
+<div class="example">
```

Comment:
Not sure how to split the responsibilities for examples between this and the [AllowShared] section. I'd say either:

- Include this mega-example in one of the two sections (I can see arguments for either), and make the other section link to the example.
- Have this section do no-xattr, [AllowResizable], and [AllowShared, AllowResizable]. Have the [AllowShared] section do no-xattr, [AllowShared], and [AllowShared, AllowResizable].

The first sounds a bit cleaner and less work, so I guess I favor that.

Reply:
Since the extended attributes are alpha-sorted, I think I'll move the mega-example down to the [AllowShared] section and link to it from the [AllowResizable] section.